programming.md

```diff
@@ -1,29 +1,29 @@
 # Programming And Software Development
 
-This file contains resources to learn programming and software development. This file is divided into 2 parts - Goal-Oriented and General Purpose. 
+This file contains resources to learn programming and software development. This file is divided into 2 parts - Goal-Oriented and General Purpose.
 
-1. Goal-Oriented resources teach you to deliver an output like making a website or an app. 
+1. Goal-Oriented resources teach you to deliver an output like making a website or an app.
 2. General Purpose resources teach you a particular language like Python or JavaScript.
 
 A Goal-Oriented listing has higher priority so if a resource falls in both general purpose and goal-oriented, it automatically goes in goal-oriented and is not duplicated.
 
 **Goal-Oriented**
 
-* [Android Development](#android-development)
-* [Web Development](#web-development)
+- [Android Development](#android-development)
+- [Web Development](#web-development)
+- [Unity 3D](#unity-3d)
```

Comment:
If we had to put this in the Goal-Oriented section, we would be naming it `Game Development`. But it looks like this resource covers only some parts of Unity and not game dev topic as a whole. So I think we can move `Unity 3D` to the General Purpose section.